Form/Type/MediaType.php

```diff
@@ -77,14 +76,34 @@ public function buildView(FormView $view, FormInterface $form, array $options)
 
     /**
      * {@inheritdoc}
+     *
+     * @deprecated To be removed when bumping to Symfony 2.7+, use MediaType::configureOptions() instead
      */
     public function setDefaultOptions(OptionsResolverInterface $resolver)
     {
+        if ($resolver instanceof OptionsResolver) {
```

Comment:
What should it prevent? I don't understand at all.

In both case (`configureOptions` call or `setDefaults` on `setDefaultOptions`) does **exactly the same thing**.

This approach does not add anything more instead of redundant code AFAIK.